.github/workflows/ci-conda-workflow.yml

```
@@ -119,16 +117,11 @@ jobs:
       run: |
         source ${CONDA}/etc/profile.d/conda.sh
         conda activate build
-        source .github/scripts/setup_ds9.sh
+        if [ "$RUNNER_OS" != "macos" ]; then
```

```suggestion
        if [ "$RUNNER_OS" != "macOS" ]; then
```
I think this is case sensitive here.

---

argle bargle arghhhhhhh ;-)

---

Definitely fixed now .....